src/com/goide/GoAnnotator.java

```diff
@@ -133,6 +137,12 @@ private static TextAttributesKey getColor(GoFieldDefinition o) {
               : GoSyntaxHighlightingColors.STRUCT_LOCAL_MEMBER;
   }
 
+  private static TextAttributesKey getColor(GoFieldNameImpl o) {
```

Comment:
I need this particular one because I don't know how to add a ```isPublic()``` method to ```GoFieldName``` or one of the interfaces that is implemented by it so that it returns ```StringUtil.isCapitalized(o.getName())```.